src/core_plugins/kibana/public/visualize/editor/agg_params.js

```diff
@@ -81,15 +81,18 @@ uiModules
         // build collection of agg params html
         type.params.forEach(function (param, i) {
           let aggParam;
+          let fields;
           // if field param exists, compute allowed fields
           if (param.name === 'field') {
-            $aggParamEditorsScope.indexedFields = getIndexedFields(param);
+            fields = $aggParamEditorsScope.indexedFields = getIndexedFields(param);
+          } else if (param.type === 'field') {
+            fields = $aggParamEditorsScope[param.name] = getIndexedFields(param);
```

Comment:
I want to have more than one input of type `field`, each with a different filtering spec (e.g., one where only numbers are shown, the other only strings)